docs/ref/utils.txt

```diff
@@ -954,6 +954,18 @@ appropriate entities.
       you can use :func:`localtime` to convert to a time in the current
       time zone.
 
+.. function:: localdate()
+
+    .. versionadded:: 1.9
+
+    Returns a :class:`~datetime.date` that represents the current date. Exactly
```

Comment:
Could this be simplified to: "Returns the current date in the current timezone if USE_TZ is True, or `date.today()` otherwise."